TODO.md

```diff
@@ -34,6 +34,8 @@
 
 ## investigate:
 
+* with pssh logging disabled, ensure we have some means of prefixing output with IP addresses
```

Comment:
if this is opt-out then there needs to be some option in the env to do so, possibly part of `hide` ...? now we see where Fabric lost it down the rabbit hole ...

if this is opt-in, then perhaps something like:

```python
with settings(output_prefix="{host}: {output}"):
    remote("...")
```

Reply:
another PR